Modules/MediaCast/classes/class.ilObjMediaCastGUI.php

```diff
@@ -730,6 +736,15 @@ private function updateMediaItem($mob, &$mediaItem)
             $mob_dir = ilObjMediaObject::_getDirectory($mob->getId());
             if (!is_dir($mob_dir)) {
                 $mob->createDirectory();
+            } else {
+                global $DIC;
```

Comment:
I suggest to move the `$DIC` to the constructor and add a new class property for `$DIC->filesystem()->web()`;